.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,8 +21,8 @@
 
 ### Related issues, RFCs
 
-<!--- Add here the number to the Github Issue or RFC that is related to this PR. -->
-<!-- **Issue number:** #123 -->
```

Comment:
This line was interpreted by the workflow as an actual issue number. The workflow was then trying to assign a label to it but since it doesn't exist it would fail.